include/RAJA/index/IndexSetBuilders.hpp

```diff
@@ -53,7 +53,7 @@ namespace RAJA
  *
  ******************************************************************************
  */
-void buildIndexSetAligned(
+void RAJASHAREDDLL_API buildIndexSetAligned(
```

Comment:
What is `RAJASHAREDDLL_API` needed?

Reply:
Some of the tests for windows shared lib builds needed this otherwise it can't find the exported symbols.